lib/module.js

```diff
@@ -243,6 +249,28 @@ export default async function SentryModule (moduleOptions) {
       logger.info('Enabling uploading of release sourcemaps to Sentry')
     })
   }
+
+  async function initSentry () {
```

Comment:
Including functions within other functions should be avoided as much as possible cause this makes them impossible to unit test. Just use function arguments and trust that the GC is smart enough to release the ref once the fn isnt needed anymroe